main.py

```diff
@@ -1,17 +1,51 @@
 import animate
 import pygame
 from pygame.locals import *
+from menu import *
+from TicTacToe import *
 
 class App:
     def __init__(self):
+        # Flag to indicate if the whole game/software is running
         self._running = True
+        # Flag to indicate if the game (after you click "start  game") is playing
+        self.playing = False
         self._clock = pygame.time.Clock()
 
         self._display_surf = None
         self.size = self.width, self.height = 640, 400
         self.FPS = 60
 
-        self.avatar = None
+        self.imidx = 0
+        self.imidy = 0
+        self.UP_KEY, self.DOWN_KEY, self.START_KEY, self.BACK_KEY = False, False, False, False
+        self.font_name = pygame.font.get_default_font()
+        #pygame.font.init()
+        #self.font_name = pygame.font.SysFont("arial",20)
+        self.BLACK, self.WHITE = (0,0,0), (255,255,255)
+        self.display = pygame.Surface((self.width,self.height))
+        self.main_menu = MainMenu(self)
```

Comment:
The self attribute is not needed when creating an object.
```suggestion
        self.main_menu = menu.MainMenu()
```